draft-ietf-quic-transport.md

```diff
@@ -439,12 +439,8 @@ Google QUIC Demultipexing Bit:
 
 Reserved:
 
-: The sixth bit (0x4) of octet 0 is reserved for experimentation.
-
-Short Packet Type:
-
-: The remaining 2 bits of octet 0 include one of 4 packet types.
-  {{short-packet-types}} lists the types that are defined for short packets.
+: The sixth, seventh, and eighth bits (0x7) of octet 0 are reserved for
+experimentation.
```

Comment:
I don't personally object to this, but does this have consensus in the WG?

Reply:
From @mnot's message "Spin Bit -- A Path Forward":
> - The transport draft editors will assure that at least three bits are available for experimentation in the next draft.

I don't think there's consensus yet that the Type field will go away, but the design direction of the protocol so far does not seem to point toward additional path-visible optional parts or forms for the short header, meaning these bits become useless once they're subsumed into the packet number encoding. This seemed like the lower-energy way to get to three bits than working around STUN and GUIC multiplexing.

Reply:
SGTM

Reply:
I think that this is a fine thing.  Removing type is good; reserving is good.

I would more simply state this as "these bits are reserved".  We might ultimately decide that the spin bit experiment is an experiment no longer.  At that point we might assign them to the spin bit(s) or find an entirely different use for those bits.

Reply:
I'd still lean toward "reserved for experimentation", with the understanding that this language is necessarily transient and will collapse one way or another before V1 final.

One, because that's the current state and the outcome of the discussion in London; two, because given the WG's love of grease to date, simply stating "this is reserved" without discussion of how it's greased would seem odd.